src/app.css

```
@@ -36,15 +36,19 @@
 	font-family: 'GeistMono';
-	src: url('/GeistMono[wght].ttf') format('truetype-variations');
+	src:
+		url('/GeistMono[wght].woff2') format('woff2-variations'),
+		url('/GeistMono[wght].ttf') format('truetype-variations');
 	font-weight: 100 900;
 	font-style: normal;
 	font-display: swap;
 }
 
 @font-face {
 	font-family: 'Geist';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)